mava/systems/jax/ippo/components.py

```diff
@@ -67,6 +68,7 @@ def on_building_init_end(self, builder: SystemBuilder) -> None:
             None.
 
         """
+        builder.store.manager_pid = os.getpid()
```

Comment:
Yeah it is not related to IPPO only I will move it in `on_building_init_start` since it is a common predefined method!

Reply:
It can stay in the hook `on_building_init_end`, but just thought we might want to include it in a more general component, than keeping it inside a component defined just for ippo. For example, instead of putting it in `ExtrasLogProbSpec` component, we add it to a more general component that will be used by other systems as well, e.g. the `Distributor` component. Or something like that, where it makes sense.

Reply:
That make sense, okay I will thanks!

Reply:
Done now it is in the `mava/components/building/environments.py` on_building_init_start method